### PR TITLE
Add language file for Romanian

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@SalmonGER for my salami <3

File: resources/config.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 # Supported Languages:
 # English (English) = eng,
+# Romana (RO) = ro,
```
Comment (SalmonDE) [inline resources/config.yml]: Can you do like this:
English (English) = eng,
German (Deutsch) = deu,
Romanian (Romana) = rom

**Comment by SalmonDE** [comment]
Love it :+1: 
Can you rename ro.ini to rom.ini?

**Comment by ghost** [comment]
yes ! >3

**Comment by SalmonDE** [comment]
Well I'll just do the changes then :+1:

**Pull request merged**